SQL/Archive/16.1/2016-08-03-FixAccessProfilePermissions.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM LorisMenuPermissions WHERE MenuID=(SELECT ID FROM LorisMenu WHERE Label='Access Profile' AND Parent=1) AND PermID=(SELECT PermID FROM permissions WHERE code='data_entry');
```

Comment:
This will make the Menu item visible to everyone.
I think you should add a row for 'access_all_profiles'